app/test_main.py

```diff
@@ -1,3 +1,61 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+    ]
+)
+def test_first_15_years(cat_age: int, dog_age: int, result: list) -> None:
+    assert (
+        get_human_age(cat_age, dog_age) == result
+    ), "first 15 cat or dog years give 1 human year"
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+    ]
+)
+def test_next_9_years_gives_second_year(
+        cat_age: int, dog_age: int, result: list
+) -> None:
+    assert (
+        get_human_age(cat_age, dog_age) == result
+    ), "the next 9 dog/cat years after first 15 give 1 more human year"
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (28, 28, [3, 2])
+    ]
+)
+def test_every_4_last_years_for_cat(
+        cat_age: int, dog_age: int, result: list
```

Comment:
It would be better to place each argument on a new line to improve the readability.